controllers/admin/AdminImportController.php

```diff
@@ -1756,39 +2362,54 @@ protected function productImportOne($info, $default_language_id, $id_lang, $forc
+                            && $categoryToCreate->add()
+                        ) {
+                            $categoryToCreate->link_rewrite = AdminImportController::createMultiLangField(
+                                $categoryLinkRewrite
+                            );
+                            $product->id_category[] = (int)$categoryToCreate->id;
                         } else {
                             if (!$validateOnly) {
-                                $this->errors[] = sprintf(
-                                    $this->trans('%1$s (ID: %2$s) cannot be saved', array(), 'Admin.Advparameters.Notification'),
-                                    $category_to_create->name[$default_language_id],
-                                    (isset($category_to_create->id) && !empty($category_to_create->id))? $category_to_create->id : 'null'
-                                );
+                                $this->errors[] = $this->trans('%1$s (ID: %2$s) cannot be saved', array(
+                                    $categoryToCreate->name[$defaultLanguageId],
+                                    !empty($categoryToCreate->id) ? $categoryToCreate->id : 'null',
+                                ), 'Admin.Advparameters.Notification');
                             }
-                            if ($field_error !== true || isset($lang_field_error) && $lang_field_error !== true) {
-                                $this->errors[] = ($field_error !== true ? $field_error : '').(isset($lang_field_error) && $lang_field_error !== true ? $lang_field_error : '').
-                                    Db::getInstance()->getMsgError();
+                            if ($fieldError !== true || isset($langFieldError) && $langFieldError !== true) {
+                                $this->errors[] = ($fieldError !== true ? $fieldError : '')
+                                    . (isset($langFieldError) && $langFieldError !== true ? $langFieldError : '')
+                                    . Db::getInstance()->getMsgError();
                             }
                         }
                     }
                 } elseif (!$validateOnly && is_string($value) && !empty($value)) {
-                    $category = Category::searchByPath($default_language_id, trim($value), $this, 'productImportCreateCat');
+                    $category = Category::searchByPath(
+                        $defaultLanguageId,
+                        trim($value),
+                        $this,
+                        'productImportCreateCat'
+                    );
                     if ($category['id_category']) {
                         $product->id_category[] = (int)$category['id_category'];
                     } else {
-                        $this->errors[] = $this->trans(
-                            '%s cannot be saved',
+                        $this->errors[] = $this->trans('%s cannot be saved',
```

Comment:
Same comment about translation placeholder. Please use a variable name surrounded by `%%` an add it in the second parameter.